README.md

```diff
@@ -1,4 +1,17 @@
 Fluo Tour
 ---------
 
+The git repository provides a barebones Maven+Java environment for the [Fluo Tour][tour].  As you go
+through the Tour edit [Main.java] and then use the following command to execute Main. This command
```

Comment:
could be `use the following command to get all of the correct dependencies on the classpath and execute Main`